include/ck/tensor_operation/gpu/grid/gridwise_2d_reduction_multiblock.hpp

```diff
@@ -244,7 +244,7 @@ struct GridwiseReduction_mk_to_m_multiblock
 
         if(thread_k_cluster_id == 0)
         {
-            if(block_group_size == 0 && !float_equal_zero{}(beta))
+            if(!float_equal_zero{}(beta))
```

Comment:
Should we remove it also here include/ck/tensor_operation/gpu/grid/gridwise_2d_multiple_reduction_multiblock.hpp ?

Reply:
I think that yes, I'll add that